server/schedule/checker_controller.go

```diff
@@ -70,11 +73,26 @@ func (c *CheckerController) CheckRegion(region *core.RegionInfo) []*operator.Ope
 
 	if c.opts.IsPlacementRulesEnabled() {
 		if op := c.ruleChecker.Check(region); op != nil {
+			added := false
+			// ignore spilt or merge
+			if op.Kind()&operator.OpReplica != 0 {
+				missPeers, count := c.GetRuleChecker().GetMissPeer(region)
+				// miss replicate more than majority should has high priority.
+				// if less than majority, it will add waiting
+				if missPeers > count/2 {
+					c.missRegionQueue.Push(missPeers, region.GetID())
```

Comment:
I think we need to add the miss peer region which op may not be added to the `WaitingOperator`

Reply:
yes, I ues count/2 to be condition to judge putting , eg regionA has one miss peer, it will not put to priority queue. if regionA has two miss regions, it will put to the queue even if the op is limitted